safe/test/data/gisv4/exposure/pop_binary_raster_20_20.asc

```diff
@@ -0,0 +1,29 @@
+NCOLS 20
```

Comment:
Why did you make a copy of this layer ? I think you should use safe/test/data/exposure, no ?